modules/cloudfront_distribution/tf_module/main.tf

```diff
@@ -21,7 +21,7 @@ resource "aws_cloudfront_distribution" "distribution" {
   enabled         = true
   is_ipv6_enabled = true
   price_class     = var.price_class
-  aliases         = var.acm_cert_arn == "" ? [] : var.domains
+  aliases         = var.acm_cert_arn == "" ? [] : concat(var.domains, formatlist("*.%s", var.domains))
```

Comment:
The domains here are required to all be listed on the certificate served by the distribution, so we might not want to add wildcards for every domain by default (see https://docs.amazonaws.cn/en_us/AmazonCloudFront/latest/DeveloperGuide/CNAMEs.html#alternate-domain-names-wildcard). Could we make this optional or just have the user specify which domains should have wildcards atttached themselves?